spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -524,7 +524,7 @@ If the injection point represents a dynamically obtained instance, then the `get
 * The `isDelegate()` method returns `true` if the injection point is a decorator delegate injection point, and `false` otherwise.
 If the injection point represents a dynamically obtained instance then `isDelegate()` returns false.
 * The `isTransient()` method returns `true` if the injection point is a transient field, and `false` otherwise.
-If the injection point represents a dynamically obtained instance then the `isTransient()` method returns `true` if the `Instance` injection point is a transient field, and `false` otherwise.
+If the injection point represents a dynamically obtained instance then the `isTransient()` method returns `true` if the `Instance` injection point is a transient field, and `false` otherwise. If this injection point is declared as transient, after bean's passivation, the value will not be restored. Instance<> injection point is the preferred approach.
```

Comment:
Emily approach is the one we discussed: CDI doesn't do any magic for `transient` injected fields when a passivated bean is activated.
What did you have in mind John?